server/handlers/meshsync_handler.go

```diff
@@ -113,7 +113,7 @@ func (h *Handler) GetMeshSyncResources(rw http.ResponseWriter, r *http.Request,
 		result = result.Offset(offset)
 	}
 
-	order = models.SanitizeOrderInput(order, []string{"created_at", "updated_at", "name"})
+	order = models.SanitizeOrderInput(order, []string{"created_at", "updated_at", "kubernetes_resource_object_meta.name"})
```

Comment:
@singh1203 the request is going for sort from client but its not sorting the data.

Reply:
@MUzairS15 can you help me with what I am missing here a request for sort is successful but the sort on the server is not reflecting on the client side.

Reply:
Try to see what SQL query is getting produced using form debug / ToSQL.
What is the value for `order` after SanitizeOrderInput?